app/components/edit_resource_button.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= link_to path,
+            method: method,
+            class: 'btn btn-outline-light btn--squish btn--edit mr-lg-2',
+            title: t('resources.edit_button.tooltip'),
+            data: { toggle: 'tooltip', placement: 'bottom' } do %>
+            <%= label %>
+            <i class="material-icons">edit</i>
```

Comment:
We need to add `aria-hidden="true"` here otherwise the screen reader with pronounce "edit".

Reply:
Added to the edit component as well as the settings buttons. Making all of these changes in 2 places really underscores the need to componentize the settings button as well...

Reply:
Ha. That's funny, @nicktiberi  because I added a comment to the settings button too, but changed my mind and removed it because I thought it was out of scope. But, yes, absolutely. If we had a central "button component" that did all of that for us....